app/components/media/tag_component.rb

```diff
@@ -24,7 +24,7 @@ def call
       if SUPPORTED_MEDIA_TYPES.include?(type.to_sym)
         media_element
       else
-        render PreviewImageComponent.new(druid:, file:, type:, resource_index: @resource_iteration.index)
+        render PreviewImageComponent.new(druid:, file:, type:, size: @resource_iteration.size, resource_index: @resource_iteration.index)
```

Comment:
This got updated. I am thinking we should probably just pass the resource_iteration to the component instead of just the index.